gwakeonlan/import_ethers.py

```diff
@@ -0,0 +1,40 @@
+# @license AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
```

Comment:
gWakeOnLAN is licensed under GPL v2 and higher and this file is licensed under a different license.
I'm not entirely sure how this double licensing applies